### ISSUE TITLE
Package request: LocalSend

### ISSUE THREAD

**Issue by alexmozaidze** [void-linux/void-packages#43691]
### Package name

localsend

### Package homepage

https://localsend.org/#/

### Description

LocalSend allows for wireless file transfer with HTTPS encryption. It's a very handy tool that can serve as a replacement for WI-Fi Direct on newer *and* older phones (WI-FI Direct is hard to use on some Android devices).

Since Flutter is not available on Void repositories, it is hard to compile from source, and the AppImage version of LocalSend is quite slow and inconvenient to use.

### Does the requested package meet the package requirements?

System, Compiled

### Is the requested package released?

Yes

**Comment by mhmdanas** [comment]
> Since Flutter is not available on Void repositories, it is hard to compile from source, and the AppImage version of LocalSend is quite slow and inconvenient to use.

Since the package is a Flutter app, isn't #12749 a blocker?

**Comment by alexmozaidze** [comment]
> Predictably, it doesn't look trivial at all.
Ouch.

**Comment by Pentaphon** [comment]
> Since the package is a Flutter app, isn't #12749 a blocker?

Yep. He has to wait until the Localsend dev makes Localsend CLI without Flutter as mentioned in https://github.com/localsend/localsend/issues/491#issuecomment-1572689821

**Comment by Vinfall** [comment]
I'm not sure how Void or Alpine packaged Flutter but I guess it does not mean you can build a musl flutter app. In other words, you might also have to wait for musl-libc support in Flutter or it can only be used on Void glibc even if it gets packaged.

You can just use KDE Connect btw.
